tutorials/3ch.md

```diff
@@ -366,17 +366,17 @@ process. The `ProcessInfo` type it returns contains the local node id and a list
 registered names, monitors and links for the process. The call returns `Nothing` if the
 process in question is not alive.
 
-### Monad Transformer Stacks 
+### Monad Transformer Stacks
```

Comment:
We should add an issue (if not fixing here) to point out the risks of this approach - @qnikst can you advise the right text to put in please?